### ISSUE TITLE
Crashes the game as of 3.6

### ISSUE THREAD

**Issue by Raptor2213** [OriginalTrinity/More-Ascension-Perk-Slots#12]
As soon as you attempt to select a tradition with this mod enabled in 3.6, the game performs a CTD.

**Comment by BluebirdGreycoat** [comment]
`As soon as you attempt to select a tradition with this mod enabled in 3.6, the game performs a CTD.`

Seconding this.

In case it helps, here are the other mods I'm using:

```
Dynamic Difficulty - Ultimate Customization
Dynamic Difficulty: More Modifiers
Dynamic Political Events
Elves of Stellaris
Ethics and Civics Classic 3.6
Extended Vanilla Palette / Colors
Gigastructural Engineering & More (3.6)
Guilli's Planet Modifiers and Features
Just Star Names (Continued)
More AI Personalities
More Events Mod
More Insults
Planetary Diversity
Sci-Fi Loading Screens
Technology Ascendant
The Zenith of Fallen Empires 3.0
```

**Comment by BaalTRB** [comment]
I started a game with a few mods, one of which was More Ascension Perk Slots, and it worked fine. I could select all my traditions as normal right up until today, when, upon taking the 8th Tradition, the game crashed.

After a quick google search I guessed it was this mod, moved it to the bottom of my mod list and tried again, only to crash again.

Now here is where it gets weird. To test further, I started a new game, with ONLY this mod, gave myself some unity and attempted to take my 1st tradition tree... only to crash immediately. I tried again, this time with some UI mods in case I needed them about, but crashed instantly on taking the first tree.

I put the More Ascension Perk Slots mod at the top (above the ui mods) for the next test, and was again able to take the first 7 tradition trees. Encouraged by this, I returned to my fully modded game, this time putting Ascension perk slots up pretty far in the list, but clearly did an oops because I returned to not being able to take any trees again.

Anyway, I'll find another Ascension perk slots mod, and leave my findings here.
